initial code for day three: historical comparison

jordansread opened the pull request:
Not my final choice for viz - is placeholder for now. But getting the data in and some code to process

Commit:
ignoring NetCDF files, adding inds and remake

Commit:
plot map from raw data comparison

Commit:
greater contrasts in years

cnell-usgs reviewed (approved):
Yay 😃  Can you change the subdirectory name to `03_historical_jread/`? 
Out of curiosity, why did you select the dates 06-01 to 09-30?

cnell-usgs committed:
Merge branch 'main' into main

Commit:
folder name change from PR review

Commit:
update ignore for folder name change

Commit:
adding this Store that I think should be deleted

Commit:
merge conflict averted!

Merge branch 'main' of github.com:jread-usgs/chart-challenge-22 into main

# Conflicts:
#	.DS_Store
#	.gitignore

Commit:
conflict clean up

jread-usgs commented:
I think this is good to go now @ceenell 

The reason I used JAS months is because that is a common simplification of the summer period (e.g., is the period used [here](https://agupubs.onlinelibrary.wiley.com/doi/10.1002/2015GL066235))
